.github/workflows/test.yaml

```diff
@@ -201,10 +192,6 @@ jobs:
         if: always()
         run: php vendor/bin/psalm --show-info=true
 
-      - name: Psalter
```

Comment:
Is there a reason why you are cleaning out the test-suite?
We have been intensifying our tests/actions lately, and it's challenging, but it gives so much rest when you comply with certain linters/syntax-checkers

Reply:
You never responded to this. We would like to keep the test-suites the same throughout the org.
Other than that, this PR is fine to merge.